Automattic/jetpack
# Blocks: use UI dark color to ensure text is readable

Fixes #19285
Fixes https://github.com/Automattic/themes/issues/5036

## Proposed changes:

- This fixes an issue with the Markdown block when using a theme with dark background colors and white text.
- It also updates the Repeat Visitor block to stop using non-available colors, and use colors from the Gutenberg shared styles instead.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

1. Go to Appearance > Themes and switch to a block theme like Twenty Twenty Three.
2. Go to Appearance > Site Editor > Styles
3. Switch to a style with a dark background and white text. You can edit a style to define those colors too.
4. Go to Posts > Add New
5. Add a Markdown block. You should be able to type in some text, and you should see your text clearly (dark color on white background).
6. Add a Repeat Visitor block; the notice should be visible to you (dark text on a white background).
